axelor-account/src/main/java/com/axelor/apps/account/web/PaymentSessionController.java

```diff
@@ -219,4 +220,19 @@ public void checkIsAllInvoiceTerms(ActionRequest request, ActionResponse respons
       TraceBackService.trace(response, e);
     }
   }
+
+  @SuppressWarnings("unchecked")
+  public void removeMultiplePaymentSessions(ActionRequest request, ActionResponse response) {
+    List<Long> paymentSessionIds = (List<Long>) request.getContext().get("_ids");
+    try {
+      int deletedSessions =
+          Beans.get(PaymentSessionService.class).removeMultiplePaymentSessions(paymentSessionIds);
```

Comment:
If paymentSessionIds is null or empty, we'll get an exception
Please add an error message if paymentSessionIds is null or empty, like "You need to select at least one payment session". (translation: "Vous devez sélectionner au moins une saisie de paiement")
You can add this message in this controller or in action-payment-session-validate-remove-multiple-payment-session, as you wish

Reply:
Added on action-validate